docs/README.md

```diff
@@ -754,3 +754,32 @@ spec:
   additionalManifests:
     name: additional-manifests
 ```
+
+## Patching provider manifests
```

Comment:
Suggestion: I would suggest extending a bit this documentation with an example of what has to be done in order to patch the manifest, because reading through the change it is obvious but probably not when just reading the below notes for the first time 🙂